README.md

```diff
@@ -1 +1,55 @@
-Review Past project
\ No newline at end of file
+# Review Past Project
+
+> This is a code review practice to test my code reviewing skills.
+
+## Built With
+
+- HTML
+- CSS
+- Linters
+- Git
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+### Setup
+
+Clone the repository and change the directory to blog
+
+```
+  $ git clone git@github.com:Favourezeugwa/Review-past-project.git
+
+  $ cd Review-past-project
+```
+
+### Install
+
+Install all project dependencies by running the command below
+
+### Run the application
+
+
+### Visit the homepage
+
+
+### Check linter errors
+
```

Comment:
I believe readme's are incomplete without having detailed steps at which a user can setup and run a website. Often times, this is usually done by adding some simple code snippets that he/she can run locally on their machine. 

What do you think about implementing this suggestion? Personally, I think these steps will enable your website to be up and running on a user's machine.